include/model.php

```diff
@@ -92,6 +92,10 @@ public function get_languages_list( $args = array() ) {
 
 				if ( ! empty( $languages ) && ! empty( $term_languages ) ) {
 					foreach ( $languages as $k => $v ) {
+						if ( ! isset( $term_languages[ 'pll_' . $v->slug ] ) ) {
+							unset( $languages[ $k ] );
+							continue;
+						}
```

Comment:
I propose to use a cleaner approach to avoid the `unset()` by not using the same variable name for the initial `WP_Term` array and the final `PLL_Language` array. Something that would finally look like:
```php
foreach ( $post_languages as $k => $v ) {
		$languages[ $k ] = new PLL_Language( $v, $term_languages[ 'pll_' . $v->slug ] );
}
```